plugins/modules/discovery_intent.py

```diff
@@ -204,13 +211,6 @@
                     - Requires valid SSH credentials to work.
                     - Avoid standard ports like 22, 80, and 8080.
                 type: str
-      use_global_cred:
```

Comment:
use_global_cred has been renamed to use_global_credentials. This too should be documented, at least in the `notes` section of the module.

Reply:
Taken care.